### PR TITLE
util-linux 2.39

### PR DESCRIPTION
**Pull request by carlocab**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/util-linux.rb [hunk 1]
```diff
@@ -49,12 +49,6 @@ class UtilLinux < Formula
     depends_on "libtool" => :build
     depends_on "pkg-config" => :build
     depends_on "gettext" # for libintl
-
-    # Fix lib/procfs.c:9:10: fatal error: 'sys/vfs.h' file not found
-    patch do
-      url "https://github.com/util-linux/util-linux/commit/3671d4a878fb58aa953810ecf9af41809317294f.patch?full_index=1"
-      sha256 "d38c9ae06c387da151492dd5862c58551559dd6d2b1877c74cc1e11754221fe4"
-    end
```
Comment (carlocab) [inline Formula/util-linux.rb:57]: Most of the block above this needs to go too.

**Comment by carlocab** [comment]
Some failures are likely related. e.g. `spaceman-diff`. Not sure if regression in `util-linux` or just tests that need updating.

**Comment by iMichka** [comment]
Fix for dump1090-mutability: #131622

**Comment by iMichka** [comment]
Fix for spaceman-diff: #131623

**Comment by iMichka** [comment]
Let's re-run this. We should be good to merge:

2023-05-18T14:42:46.9985746Z Warning: 2 failed steps ignored!
2023-05-18T14:42:46.9986927Z brew linkage --cached --test --strict darkice -> FIXED
2023-05-18T14:42:46.9987456Z brew linkage --cached --test --strict dirt -> FIXED
2023-05-18T14:42:46.9987779Z Error: 11 failed steps!
2023-05-18T14:42:46.9988138Z brew test --retry --verbose dump1090-mutability -> FIXED
2023-05-18T14:42:46.9988492Z brew test --retry --verbose kafka -> NOT REPRODUCIBLE LOCALLY
2023-05-18T14:42:46.9989472Z brew test --retry --verbose monero -> FIXED
2023-05-18T14:42:46.9989987Z brew test --retry --verbose mvnvm -> NOT REPRODUCIBLE LOCALLY
2023-05-18T14:42:46.9990298Z brew test --retry --verbose qcli -> NOT REPRODUCIBLE LOCALLY
2023-05-18T14:42:46.9990868Z brew test --retry --verbose questdb -> NOT REPRODUCIBLE LOCALLY
2023-05-18T14:42:46.9991294Z brew linkage --test root -> KNOWN TO BE BROKEN
2023-05-18T14:42:46.9991699Z brew test --retry --verbose solr -> NOT REPRODUCIBLE LOCALLY
2023-05-18T14:42:46.9992019Z brew test --retry --verbose sonarqube -> NOT REPRODUCIBLE LOCALLY
2023-05-18T14:42:46.9992342Z brew test --retry --verbose spaceman-diff -> FIXED
2023-05-18T14:42:46.9992658Z brew test --retry --verbose walkmod -> TEST BROKEN BUT DEPRECATED

**Review by iMichka — APPROVED** [review]

**Pull request merged**